package.json

```diff
@@ -38,7 +40,7 @@
     "typescript": "^4.4.4"
   },
   "dependencies": {
-    "axios": "1.1.3"
+    "axios": "^1.1.3"
```

Comment:
`axios` experienced a major regression in version `1.2.0`, however I don't mind upgrading but I'd prefer if this was explicitly pinned to `1.3.4`. Thank you!

Reply:
I am OK with this change as I have done all of my testing with axios v1.3.4. I believe I have submitted a correction.